validator/src/main.rs

```diff
@@ -1360,6 +1360,13 @@ pub fn main() {
                 .help("After processing the ledger and the next slot is SLOT, wait until a \
                        supermajority of stake is visible on gossip before starting PoH"),
         )
+        .arg(
+            Arg::with_name("no_wait_for_vote_to_start_leader")
```

Comment:
might be good to add a warning here that if > 33% of the cluster goes down and restarts, everyone will need to set this flag, otherwise even on restart nobody will make progress.

Reply:
If 33% goes down, we have to do a manual restart with `--wait-for-supermajority` flag, right? In that we detect that we had to wait for supermajority and we skip this check for the vote. So, it shouldn't be necessary to use it in that case. The only case we need it is if you are starting a bootstrap leader(s) without `--wait-for-supermajority`